token/services/interop/htlc/db/sql/tokendbext.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package sql
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"runtime/debug"
+
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/db/driver"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/interop/htlc"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/logging"
+	"github.com/pkg/errors"
+)
+
+var logger = logging.MustGetLogger("token-sdk.sql.htlc")
+
+type htlcTokenDBExtensionTables struct {
+	Tokens string
+}
+
+type HTLCTokenDBExtension struct {
+	table htlcTokenDBExtensionTables
+}
+
+func NewHTLCTokenDBExtension(table htlcTokenDBExtensionTables) *HTLCTokenDBExtension {
+	return &HTLCTokenDBExtension{table: table}
+}
+
+func (e *HTLCTokenDBExtension) GetSchema() string {
+	return fmt.Sprintf(`
+		-- Tokens
```

Comment:
I guess Tokens should be 'HTLC' or 'Interop' here. Or wait, do you actually mean the existing 'tokens' table here, or a new one that exists next to the 'tokens' table?

If it's the first, that could become an issue with versioning and updates (our code doesn't handle db migrations yet, but should in the near future and extending an existing table adds another dimension that makes it harder).